tools/build_rules/maven_artifact.bzl

```diff
@@ -0,0 +1,42 @@
+load("@rules_jvm_external//:defs.bzl", _rje_artifact = "artifact")
+load("@io_bazel_rules_scala_config//:config.bzl", "SCALA_MAJOR_VERSION")
+
+def _safe_name(coord):
+    return coord.replace(":", "_").replace(".", "_").replace("-", "_")
+
+def maven_artifact(coord, repository_name = "maven"):
```

Comment:
is there anything here that could be substituted with an existing bazel integration? eg.. https://github.com/bazel-contrib/rules_jvm_external?tab=readme-ov-file

Reply:
i did not find matching one from https://github.com/bazel-contrib/rules_jvm_external/blob/master/specs.bzl